### DISCUSSION TITLE
Moonraker update manager

### DISCUSSION THREAD

**Discussion by Caouette1988** [mainsail-crew/spyglass#44] [Ideas]
Hi, would it be possible to make spyglass updated by moonraker?

I try to add these line to MOONRAKER.CONF :
[update_manager spyglass]
type: git_repo
primary_branch: main
path: ~/spyglass
origin: https://github.com/roamingthings/spyglass.git
managed_services: spyglass

Then moonraker return this notification : 
Moonraker warning
[update_manager spyglass]: Moonraker is not permitted to restart service 'spyglass'. To enable management of this service add spyglass to the bottom of the file /home/pi/printer_data/moonraker.asvc. To disable management for this service set 'is_system_service: False' in the configuration for this section.

I've done this, and moonraker return no error. 
Do you think this method would work or something is missing?
Should I add a install_script command in the [update_manager spyglass] ?

**Comment by roamingthings** [comment]
I have not looked into how the update manager works. However, I think this is an important feature. But I cannot tell you when I've got the time to get to this.

**Reply by Caouette1988** [reply]
I'll give you feedback if it work at your next update.

**Comment by roamingthings** [comment]
I've added an FR for this https://github.com/roamingthings/spyglass/issues/45

**Comment by mryel00** [comment]
#45 got merged so closing this
